source/administration/identity-access-management/pluggable-authorization.rst

```diff
@@ -0,0 +1,132 @@
+.. _minio-external-access-management-plugin:
+
+=======================================
+MinIO External Access Management Plugin
+=======================================
+
+.. default-domain:: minio
+
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+The MinIO Access Management Plugin provides a REST interface for offloading authorization through a webhook service.
+
+Once enabled, MinIO sends the request and credential details for every API call to the configured external HTTP(S) endpoint and looks for a response of ALLOW or DENY.
+MinIO can therefore delegate the access management to the external system instead of relying on S3 :ref:`policy based access control <minio-policy>`.
+
+Configuration Settings
+----------------------
+
+You can configure the MinIO External Access Management Plugin using the following environment variables or configuration settings.
+
+.. tab-set::
+
+   .. tab-item:: Environment Variables
+
+      Specify the following :ref:`environmental variables <minio-server-envvar-external-access-management-plugin>` to each MinIO server in the deployment:
+
+      .. code-block:: shell
+         :class: copyable
+
+         MINIO_POLICY_PLUGIN_URL="https://external-authz.example.net:8080/authz"
+
+         # All other envvars are optional
+         MINIO_POLICY_PLUGIN_AUTH_TOKEN="Bearer TOKEN"
+         MINIO_POLICY_PLUGIN_ENABLE_HTTP2="OFF"
+         MINIO_POLICY_PLUGIN_COMMENT="External Access Management using PROVIDER"
+
+   .. tab-item:: Configuration Settings
+
+      Set the following configuration settings using the :mc-cmd:`mc admin config set` command:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin config set policy_plugin \
+            url="https://external-authz.example.net:8080/authz" \
+
+            # All other config settings are optional
+            auth_token="Bearer TOKEN" \
+            enable_http2="off" \
+            comment="External Access Management using PROVIDER"
+
+Authentication and Authorization Flow
+-------------------------------------
+
+The login flow for an application is as follows:
+
+1. The client includes authentication information as part of performing the API call
+
+2. The configured Identity Manager authenticates the client
+
+3. MinIO makes a POST call to the configured access management plugin URL which includes the context of the API call and authentication data
+
+4. On successful authorization, the access manager returns a ``200 OK`` response with a JSON body that matches either  of the following:
```

Comment:
```suggestion
4. On successful authorization, the access manager returns a ``200 OK`` response with a JSON body of either ``result true`` or ``"result" : { "allow" : true }``:
```

Then delete lines 70-75, since we already have the example on lines 128ff.